src/test/smoke/datascience.smoke.test.ts

```diff
@@ -3,50 +3,28 @@
 
 'use strict';
 
-import { assert } from 'chai';
 /* eslint-disable , no-invalid-this, @typescript-eslint/no-explicit-any */
 
 import * as fs from 'fs-extra';
 import * as path from 'path';
 import * as vscode from 'vscode';
-import { ISystemPseudoRandomNumberGenerator } from '../../client/datascience/types';
-import { IExtensionTestApi, openFile, setAutoSaveDelayInWorkspaceRoot, waitForCondition } from '../common';
+import { openFile, setAutoSaveDelayInWorkspaceRoot, waitForCondition } from '../common';
 import { EXTENSION_ROOT_DIR_FOR_TESTS, IS_SMOKE_TEST } from '../constants';
 import { sleep } from '../core';
-import { closeActiveWindows, initialize, initializeTest } from '../initialize';
+import { closeActiveWindows, initializeTest } from '../initialize';
 
 const timeoutForCellToRun = 3 * 60 * 1_000;
 suite('Smoke Tests', () => {
-    let api: IExtensionTestApi;
     suiteSetup(async function () {
         if (!IS_SMOKE_TEST) {
             return this.skip();
         }
-        api = await initialize();
         await setAutoSaveDelayInWorkspaceRoot(1);
     });
     setup(initializeTest);
     suiteTeardown(closeActiveWindows);
     teardown(closeActiveWindows);
 
-    test('Random bytes generation', async function () {
-        return this.skip(); // Failing on windows. Tracked by 4444
-        // We do have a unit test testing this, however create a smoke test to
-        // ensure that the bundling of the native node modules worked
```

Comment:
Does this not still apply? It's verifying the bundling worked.

Reply:
Oh, ok, i thought we had unit tests and that was sufficient.

Reply:
Will revert.